api_proxy/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# How to build:
+# docker run --rm -e BUILD_GOOS="linux" -e BUILD_GOARCH="amd64" -v $(pwd):/src \
+#   -v /var/run/docker.sock:/var/run/docker.sock \
+#   centurylink/golang-builder-cross gcr.io/$DOCKER_NAMESPACE/api-proxy
```

Comment:
The cross compiler is so someone can replicate our builds on any system (including not amd64 linux).
I like go2docker except the fact it needs to be built separately by all users and it doesn't do deps.